src/openfermion/testing/circuit_validation_test.py

```diff
@@ -0,0 +1,55 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+'''Tests for circuit_validation.py'''
+
+import cirq
+from openfermion import (
+    QubitOperator,)
+
+from openfermion.testing.circuit_validation import (
```

Comment:
Remove parentheses.

Reply:
When I do this the line is 82 characters long, how do you suggest getting around this?

Reply:
The parenthesis are for when we are importing a lot of stuff. Since we are only importing one thing here, we can do

```
from openfermion.testing.circuit_validation import \
validate_trotterized_evolution
```

Reply:
Alright, I added this.

Reply:
I guess I personally prefer avoiding backslashes when possible, but I'm happy either way...